code/game/objects/items/weapons/implant/implants/carrion/carrion_spiders.dm

```diff
@@ -11,6 +11,11 @@
 	var/gene_price = 0
 	var/do_gibs = TRUE
 	var/last_stun_time = 0 //Used to avoid cheese
+	var/ignore_activate_all = FALSE
+
+	var/assigned_group_1 = FALSE
+	var/assigned_group_2 = FALSE	
+	var/assigned_group_3 = FALSE
```

Comment:
You could make these into a bitflag, which means in the future groups can be added/removed easily.

Reply:
I'd have to look into how these work (I don't actually know how to code) and make these changes in a different PR so this one can hopefully get merged for this launch.